joomla/joomla-cms
# Language Change for issue #38659

Pull Request for Issue #38659 .

### Summary of Changes
Changed the + symbol to "then" in System Panel
Added "then" between the keys in the xml text and made it easier to understand the shortcuts

Before:
J + X  Keyboard Shortcuts

J then A Save
J then S Save & Close
J then Q Cancel 
J then N New
J then F Search 
J then O Options
J then H Help
J then M Toggle 
J then X Overview
J then D Home 

After:
J then X Keyboard Shortcuts

Enables keyboard shortcuts on the administrator site, which can be provided by other plugins and includes directly the following list of shortcuts:

J then A Apply
J then S Save & Close
J then Q Cancel (Quit)
J then N New
J then F Search (Find)
J then O Options
J then H Help
J then M Toggle **M**enu
J then X Overview / E**x**tras
J then D Home **D**ashboard



### Testing Instructions
Go to System and see that the text on the system Panel changed from:
![grafik](https://user-images.githubusercontent.com/828371/187940084-7775dbfc-cac6-408f-8202-ec37c8ca5cc3.png)
to 
![grafik](https://user-images.githubusercontent.com/828371/187946143-008f46d0-e9c6-4891-9c77-5e7da6d275a4.png)

Go to System Plugin Shortcuts and see the description changed from:
![grafik](https://user-images.githubusercontent.com/828371/187940578-1dda29d0-1c53-45db-ae81-8acb01c7d2c9.png)

to 
![grafik](https://user-images.githubusercontent.com/828371/188077705-fe5fe258-8c4d-4b63-aba7-95325ec1e706.png)

Same should be visible in the modal window when you press J then X
